packages/story-editor/src/components/canvas/karma/canvasKeyboardNavigation.karma.js

```diff
@@ -92,40 +96,97 @@ describe('Canvas - keyboard navigation', () => {
     // enter into the canvas
     await fixture.events.keyboard.press('Enter');
 
-    const {
-      state: {
-        currentPage: { elements },
-      },
-    } = await fixture.renderHook(() => useStory());
-
-    const backgroundElement = elements.find((element) => element.isBackground);
+    const selectedElements = await fixture.renderHook(() =>
+      useStory(({ state }) => state.selectedElements)
+    );
 
+    expect(selectedElements.length).toBe(1);
+    expect(selectedElements[0].isBackground).toBe(true);
     expect(document.activeElement.getAttribute('data-element-id')).toBe(
-      backgroundElement.id
+      selectedElements[0].id
     );
```

Comment:
maybe nice to add a lil comment above? took me a sec to parse why these updates changed functionality
```
// see that background element is selected & focused
```